sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/administration/package-info.java

```diff
@@ -2,9 +2,165 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing classes for creating a
- * {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationAsyncClient}
- * and {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient}
- * to perform operations on Azure Metrics Advisor service.
+ * <p><a href="https://learn.microsoft.com/azure/ai-services/metrics-advisor/">
+ * Azure Metrics Advisor</a> is a cloud-based service provided by Microsoft Azure that provides a set of APIs for
+ * data ingestion, anomaly detection, and diagnostics, without needing to know machine learning.</p>
+ *
+ * <p>The Azure Metrics Advisor client library allows Java developers to interact with the Azure Metrics Advisor
+ * service.
+ * It provides a set of classes and methods that abstract the underlying RESTful API of Azure
+ * Metrics Advisor, making it easier to integrate the service into Java applications.</p>
+ *
+ * <p>The Azure Metrics Advisor client library provides the following capabilities:</p>
+ *
+ * <ol>
+ *     <li>Analyze multi-dimensional data from multiple data sources.</li>
+ *     <li>Identify and correlate anomalies</li>
+ *     <li>Configure and fine-tune the anomaly detection model used on your data</li>
+ *     <li>Diagnose anomalies and help with root cause analysis</li>
+ *     <li>Connecting your own time series data</li>
+ *     <li>Fine tuning the anomaly detection configuration</li>
+ *     <li>Configuring alerts for detected anomalies</li>
+ * </ol>
+ *
+ * <h2>Getting Started</h2>
+ *
+ * <p>The Azure Metrics Advisor library provides advisor clients like
+ * {@link com.azure.ai.metricsadvisor.MetricsAdvisorAsyncClient}
+ * and {@link com.azure.ai.metricsadvisor.MetricsAdvisorClient} to connect to the Metrics Advisor
+ * Azure Cognitive Service to perform data monitoring.
+ * It also provides administration clients like
+ * {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient}
+ * and {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationAsyncClient} to
+ * build and manage models from custom documents.</p>
+ *
+ * <p>Service clients are the point of interaction for developers to use Azure Form Recognizer.
+ * {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationClient} is the synchronous service client and
+ * {@link com.azure.ai.metricsadvisor.administration.MetricsAdvisorAdministrationAsyncClient} is the asynchronous service client.
+ * The examples shown in this document use a credential object named DefaultAzureCredential for authentication, which is
```

Comment:
Below samples also shows the use of MetricsAdvisorKeyCredential, we should consider mentioning it here.